examples/serverless/main.tf

```diff
@@ -138,7 +162,7 @@ module "aurora_mysql_v2" {
     max_capacity = 10
   }
 
-  instance_class = "db.serverless"
+  cluster_instance_class = "db.serverless"
```

Comment:
Running into some apply errors here and similarly in the global_cluster example. I haven't been able to look too far into it yet but it's trying to set `db_cluster_instance_class` and erroring on that.